File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1553.c
#include <stdio.h>

int main(){
    
    int n, k, p;
    while(1){
        scanf("%d %d", &n, &k);
        if(n == 0 && k == 0) break;
        
        int i, perg[110] = {}, freq = 0;
        for(i = 0; i < n; i++){
            scanf("%d", &p);
            perg[p]++;
        }
        
        for(i = 1; i <= 100; i++){
            if(perg[i] >= k) freq++;
        }
        
        printf("%d\n", freq);
    }

    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1553.py
while True:
    l = [int(i) for i in input().split()]
    n = l[0]
    k = l[1]
        
    if n == k == 0: break

    perg = []
    for i in range(110):
        perg.append(0)
        
    freq = 0
    
    v = [int(i) for i in input().split()]
    for i in v:
        perg[i] += 1
    
    for i in range(1, 101):
        if perg[i] >= k: freq += 1
    
    print(freq)